conda_forge_tick/migrators/core.py

```diff
@@ -471,7 +472,9 @@ def predecessors_not_yet_built(self, attrs: "AttrsTypedDict") -> bool:
             if (
                 m_pred_json
                 and m_pred_json.get("PR", {"state": "open"}).get("state", "") == "open"
+                and not payload.get("archived", False)
```

Comment:
@CJ-Wright Here is the bug. We need to always say archived nodes in the graph are migrated. We did that in the if statement above, but not in this one.

Reply:
ok, you might refactor this to use a common var.

Reply:
done.